deploy/helm-chart/templates/_helpers.tpl

```diff
@@ -31,6 +31,14 @@ Create control plane config name.
 {{- printf "%s-config" $name | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Create data plane config name.
+*/}}
+{{- define "nginx-proxy.config-name" -}}
```

Comment:
would nginx-proxy or nginx-proxy-name suit better?

otherwise, nginx-gateway.proxy-config-name will be more consistent with the existing names  I think